static/src/javascripts/projects/common/modules/experiments/tests/adblocking-response.js

```diff
@@ -37,9 +39,11 @@ define([
         }, {
             id: 'variantA',
             test: function () {
-                //TODO check also if not a subscriber
                 detect.getFfOrGenericAdbockInstalled.then(function (adblockUsed) {
-                    if (adblockUsed && !config.page.isFront && !userFeatures.isPayingMember()) {
+                    if (adblockUsed && !config.page.isFront &&
+                        !userFeatures.isPayingMember() &&
+                        config.page.webTitle !== 'Subscriber number form' &&
```

Comment:
I am not sure if this is the best way to detect certain page. But on the other hand, it's the easiest and straightforward method.